emanote/src/Emanote/View/Feed.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Emanote.View.Feed where
+
+import Data.Aeson.Optics (key, _String)
+import Data.Time (UTCTime)
+import Data.Time.Format
+import Emanote.Model (Model)
+import Emanote.Model.Note (Note (..))
+import Emanote.Model.Query (parseQuery, runQuery)
+import Emanote.Model.Title (toPlain)
+import Emanote.Route.ModelRoute
+import Emanote.Route.R (indexRoute)
+import Emanote.Route.SiteRoute
+import Optics.Operators ((^?))
+import Optics.Optic ((%))
+import Relude
+import Text.Atom.Feed qualified as Atom
+import Text.Atom.Feed.Export qualified as Export (textFeed)
+
+getNotes :: Text -> Model -> [Note]
+getNotes tagName model = case parseQuery ("tag:" <> tagName) of
```

Comment:
Instead of hardcoding the query, we should make that configurable in the frontmatter or -- better -- automatically use the first (and only) query in the body of the Markdown note.